packages/reaction-shipping/client/templates/shipping.html

```diff
@@ -14,195 +12,138 @@
+  {{>addShippingProvider}} {{/each}}
 </template>
-
-
 <template name="addShippingMethod">
-  {{#autoForm
-    schema="ReactionCore.Schemas.ShippingMethod"
-    doc=.
-    id="shipping-method-add-form"
-    resetOnSuccess=false
-    }}
-
-    <div class="panel panel-default">
-      <div class="panel-heading">Add rate</div>
-      <div class="panel-body">
-        <div class="col-md-6">
-          {{>afQuickField name='name' autofocus=''}}
-          {{>afQuickField name='label'}}
-          {{>afQuickField name='group'}}
-          {{>afQuickField name='enabled' value=true}}
-        </div>
-        <div class="col-md-6">
-          {{>afQuickField name='cost'}}
-          {{>afQuickField name='handling'}}
-          {{>afQuickField name='rate'}}
-        </div>
-        <div class="pull-right">
-          <button type="submit" class="btn btn-primary">Submit</button>
-          <button type="cancel" class="btn btn-primary cancel">Cancel</button>
-        </div>
+  {{#autoForm schema="ReactionCore.Schemas.ShippingMethod" doc=. id="shipping-method-add-form" resetOnSuccess=false }}
+  <div class="panel panel-default">
+    <div class="panel-heading">Add rate</div>
+    <div class="panel-body">
+      <div class="col-md-6">
+        {{>afQuickField name='name' autofocus=''}} {{>afQuickField name='label'}} {{>afQuickField name='group'}} {{>afQuickField name='enabled' value=true}}
+      </div>
+      <div class="col-md-6">
+        {{>afQuickField name='cost'}} {{>afQuickField name='handling'}} {{>afQuickField name='rate'}}
+      </div>
+      <div class="pull-right">
+        <button type="submit" class="btn btn-primary">Submit</button>
+        <button type="cancel" class="btn btn-primary cancel">Cancel</button>
       </div>
     </div>
-
-
-
+  </div>
   {{/autoForm}}
 </template>
 
 
 <template name="editShippingMethod">
-  {{#autoForm
-    schema="ReactionCore.Schemas.ShippingMethod"
-    id="shipping-method-edit-form"
-    template="bootstrap3-inline"
-    doc=.
-    }}
-
-
-    <div class="panel panel-default">
-      <div class="panel-heading">Update {{name}} rate</div>
-      <div class="panel-body">
-        <div class="col-md-6">
-          {{>afQuickField name='name' autofocus=''}}
-          {{>afQuickField name='label'}}
-          {{>afQuickField name='group'}}
-          {{>afQuickField name='enabled'}}
-        </div>
-        <div class="col-md-6">
-          {{>afQuickField name='cost'}}
-          {{>afQuickField name='handling'}}
-          {{>afQuickField name='rate'}}
-        </div>
+  {{#autoForm schema="ReactionCore.Schemas.ShippingMethod" id="shipping-method-edit-form" template="bootstrap3-inline" doc=selectedMethodDoc}}
+  <div class="panel panel-default">
+    <div class="panel-heading">Update {{name}} rate</div>
+    <div class="panel-body">
+      <div class="col-md-6">
+        {{>afQuickField name='name' autofocus=''}} {{>afQuickField name='label'}} {{>afQuickField name='group'}} {{>afQuickField name='enabled'}}
```

Comment:
When you reformat an entire block like that, it makes it really hard to tell what you actually changed that's relevant to this bug. Also the line length makes it difficult to review within Github PR's.